src/app/api/import-to-sanity/route.ts

```
@@ -2,7 +2,8 @@ import { NextRequest, NextResponse } from 'next/server'
 import { createClient } from '@sanity/client'
 import fs from 'fs/promises'
 import path from 'path'
-import { MigrationRecord } from '@/types/migration'
+import { MigrationRecord, isSanityPost, getContentTitle } from '@/types/migration'
```

ESLint

[error] 'isSanityPost' is defined but never used.

(@typescript-eslint/no-unused-vars)